import { InvalidArgumentError } from '../error';
export { InvalidArgumentError };
export declare function take<T>(iterable: Iterable<T>, count: number): Iterable<T>;
